sql/api/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryListener.scala

```diff
@@ -122,25 +171,22 @@ object StreamingQueryListener extends Serializable {
   }
 
   private[spark] object QueryStartedEvent {
-    private val mapper = {
-      val ret = new ObjectMapper() with ClassTagExtensions
-      ret.registerModule(DefaultScalaModule)
-      ret.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-      ret
-    }
 
-    private[spark] def jsonString(event: QueryStartedEvent): String =
```

Comment:
Remove this  and the others because they were not used, and they led to some confusion on my end.